modules/ztp-customizing-a-managed-site-using-pgt.adoc

```diff
@@ -20,49 +20,49 @@ Use the following procedure to customize the policies that get applied to the ma
 
 .Procedure
 
-. Create a `PolicyGenTemplate` CR for site-specific configuration CRs.
+. Create a `{policy-gen-cr}` CR for site-specific configuration CRs.
 
-.. Choose the appropriate example for your CR from the `out/argocd/example/policygentemplates` folder, for example, `example-sno-site.yaml` or `example-multinode-site.yaml`.
+.. Choose the appropriate example for your CR from the `out/argocd/example/policygentemplates` folder, for example, `{policy-prefix}example-sno-site.yaml` or `{policy-prefix}example-multinode-site.yaml`.
 
 .. Change the `bindingRules` field in the example file to match the site-specific label included in the `SiteConfig` CR. In the example `SiteConfig` file, the site-specific label is `sites: example-sno`.
 +
 [NOTE]
 ====
-Ensure that the labels defined in your `PolicyGenTemplate` `bindingRules` field correspond to the labels that are defined in the related managed clusters `SiteConfig` CR.
+Ensure that the labels defined in your `{policy-gen-cr}` `bindingRules` field correspond to the labels that are defined in the related managed clusters `SiteConfig` CR.
 ====
 
 .. Change the content in the example file to match the desired configuration.
 
-. Optional: Create a `PolicyGenTemplate` CR for any common configuration CRs that apply to the entire fleet of clusters.
+. Optional: Create a `{policy-gen-cr}` CR for any common configuration CRs that apply to the entire fleet of clusters.
 
-.. Select the appropriate example for your CR from the `out/argocd/example/policygentemplates` folder, for example, `common-ranGen.yaml`.
+.. Select the appropriate example for your CR from the `out/argocd/example/policygentemplates` folder, for example, `{policy-prefix}common-ranGen.yaml`.
```

Comment:
same here : The folder here should be {argocd-folder} since it is used for PGT and ACMPG